.github/workflows/dev_on_workflow_tests_pro_e2e.yml

```
@@ -152,12 +152,12 @@ jobs:
           CYPRESS_PROJECT_ID: ${{ steps.secrets.outputs.CYPRESS_CLOUD_PROJECT_ID }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - name: "Move cypress videos"
-        if: ${{ github.ref != 'refs/heads/master' }} # useless on master bc Cypress Cloud
+        if: always() && failure() && github.ref != 'refs/heads/master' # useless on master bc Cypress Cloud
```

À mon avis, le always va tout override, non ?
```suggestion
        if: failure() && github.ref != 'refs/heads/master' # useless on master bc Cypress Cloud
```